LibNoDaveConnectionLibrary/PLCs/S7_xxx/MC7/MC7Converter.cs

```diff
@@ -189,7 +191,7 @@ internal static S7Block GetAWLBlockBasicInfoFromBlockHeader(byte[] MC7Code, int
                  * xx      = Nertworks
                  */
 
-                retBlock.BlockVersion = new Version(MC7Code[2] / 10, MC7Code[2] % 10);
+                //retBlock.BlockVersion = new Version(MC7Code[2] / 10, MC7Code[2] % 10);

```

Comment:
Can you create a pull req? I think it's the Version of the Block you see in simatic manager